src/javascripts/helpers/validationForm.js

```diff
@@ -61,8 +61,8 @@ export const validateTransaction = (data) => {
   ]
 
   for(const field of transactionFields){
-   
-    if (!data[field.field] && !data[field.field].trim()){
+    
```

Comment:
Remove this

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks !